web-app/src/pages/tournament/TournamentsList.vue

```diff
@@ -1,34 +1,39 @@
 <template>
   <div>
     <grid-list-mobile
-      :data="data"
+      :data="trueData"
       :columns="columns"
       :filter="filter"
-      :expanded="expanded"
-      :pagination="pagination"
+      :pagination.sync="pagination"
     />
     <grid-list-desktop
-      :data="data"
+      :data="trueData"
       :columns="columns"
       :filter="filter"
-      :expanded="expanded"
-      :pagination="pagination"
+      :pagination.sync="pagination"
     />
   </div>
 </template>
 
 <script lang="ts">
-import { Vue, Component } from 'vue-property-decorator';
+import { Vue, Component, Watch } from 'vue-property-decorator';
 import GridListMobile from '../../components/tournament/GridListMobile.vue';
 import GridListDesktop from '../../components/tournament/GridListDesktop.vue';
+import { Team } from '../../../../shared/types/Tournament';
+import { IPagination } from './../../components/models';
+import User from '../../../../shared/types/User';
+import api from './../../services/API/index';
+import { TournamentListData, ListUser } from './types';
 @Component({
   components: { GridListMobile, GridListDesktop },
 })
 export default class TournamentsList extends Vue {
+  private trueData: TournamentListData[] = [];
   private filter = '';
-  private expanded = '';
-  private pagination = {
-    rowsPerPage: 0,
+  private pagination: IPagination = {
+    page: 1,
+    rowsPerPage: 1,
+    rowsNumber: 0,
   };
```

Comment:
I think it may not have a type `IPagination`. Object got a type after assignment

Reply:
Bad thinking

Reply:
What do you mean?  TS/Vue should infer a type from assignment